scc/git.py

```diff
@@ -4084,6 +4084,7 @@ class BumpVersionConda(GitRepoCommand):
     SUFFIX = " %}"
     KEY_NAME = "name"
     KEY_VERSION = "version"
+    KEY_GIT_VERSION = "git_version"
```

Comment:
Assuming it's meant to be 
```suggestion
    KEY_GIT_VERSION = "github_version"
```